### DISCUSSION TITLE
Ender-3 S1 Pro ?

### DISCUSSION THREAD

**Discussion by BalooDK** [mriscoc/Ender3V2S1#251] [General]
Does this fantastic firmware work on the S1 Pro with a touch screen ?

**Comment by hayden-t** [comment]
works with v2 screen, cheap and easy swap

**Reply by BalooDK** [reply]
Thanks , I just ordered a v2 screen kit.
Will it just work, or do I have change anything before flashing pro firmware ?

**Reply by hayden-t** [reply]
it just works, you need a longer cable than what it comes with. about 65cm 10 pin idc

**Reply by BalooDK** [reply]
Great. 
Thank you

**Reply by Double077** [reply]
Hello, where did u order the conversion kit from ? The s1 pro touch screen is pure garbage tbh + the Creality frimware sucks cant use m600 or m25 it just freezes the whole thing

**Reply by BalooDK** [reply]
I ordered from a local shop in my country (Denmark). Just search for a V2 screen. The cable that came with the V2 screen is just long enough. It’s very easy to switch it. 
Screen was blank until I updated to the s1 firmware.

**Reply by hayden-t** [reply]
aliexpress $25 just get or make a longer 10p idc cable, or make do with the one it comes with

**Comment by Squigie16** [comment]
Will there be plans on releasing the Firmware for touch screens?

**Reply by Double077** [reply]
that would be lovely

**Reply by hayden-t** [reply]
Dev has stated he has no plans to support touch screen
